examples.livemd

```diff
@@ -528,6 +533,34 @@ Boombox.run(input: "rtsp://localhost:#{rtsp_port}/", output: "#{out_dir}/index.m
 
 <!-- livebook:{"branch_parent_index":0} -->
 
+## Receive RTP, broadcast via HLS
+
+To receive the stream, visit http://localhost:1234/hls.html after running the cell below
+
+```elixir
+rtp_port = 50001
+t =
+  Task.async(fn ->
+    Boombox.run(
+      input: {:rtp, port: rtp_port, track_configs: [audio: [encoding: :OPUS], video: [encoding: :H264]]},
```

Comment:
WDYT about making it more flat, meaning 
```suggestion
      input: {:rtp, port: rtp_port, audio_encoding: :OPUS, video_encoding: :H264},
```

Reply:
it would work for this specific case, but not if we want to pass clock rate or payload type

Reply:
Maybe something like 
```
input: {:rtp, port: rtp_port, audio_config: [encoding: :OPUS], video_config: [encoding: :H264]}
```
would be a good middleground

Reply:
Not sure how it would look like in the CLI interface

Reply:
and how would the least nested option look? 
```bash
-i --rtp --port rtp_port --audio_encoding OPUS --video_encoding H264
```
?

Reply:
yup

Reply:
yeah, I don't think any nesting is feasible in CLI when using only -- args, but in Elixir API I think it adds clarity. I guess these are possible approaches that occur to me:
- the "flat" version - nicer CLI, not so nice Elixir API
- the single nest version - nicer Elixir API, would require us to rethink the CLI
- both - have both versions be available, allow for both `audio_clock_rate: 48_000` and `audio_config: [clock_rate: 48_000]`. This creates a bit of ambiguity, but could be reasonable.
what do you think?

Reply:
Personally, I'd go for the flat version and I'm definitely against having both versions at once :P If we go for a nested version, I guess we should rethink the whole API, for example https://github.com/membraneframework/boombox/blob/master/lib/boombox.ex#L12